### PR TITLE
Issue/nighttime zeros

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

make sure solar generation is 0 at nighttime

Add a new function [here](https://github.com/openclimatefix/india-forecast-app/blob/issue/nighttime-zeros/india_forecast_app/models/pvnet/utils.py#L110) and [here](https://github.com/openclimatefix/india-forecast-app/blob/issue/nighttime-zeros/india_forecast_app/models/pvnet/model.py#L103)
added a test [here](https://github.com/openclimatefix/india-forecast-app/blob/issue/nighttime-zeros/tests/models/pvnet/test_utils.py#L8)

Sorry, there's a fair amount of formatting changes in this PR

#64 

## How Has This Been Tested?

CI tests + added a new test
- [ ] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: india_forecast_app/app.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def save_forecast(db_session: Session, forecast, write_to_db: bool):
 )
 @click.option(
     "--log-level",
-    default="info",
+    default="debug",
```
Comment (peterdudfield) [inline india_forecast_app/app.py:224]: remove

File: india_forecast_app/models/pvnet/model.py [hunk 2]
```diff
@@ -123,44 +126,65 @@ def predict(self, site_id: str, timestamp: dt.datetime):
             ]
         )  # index 3 is the 50th percentile)
 
-
```
Comment (peterdudfield) [inline india_forecast_app/models/pvnet/model.py:126]: below this is just all formating, by running blacks

File: india_forecast_app/models/pvnet/utils.py [hunk 3]
```diff
@@ -97,17 +93,46 @@ def process_and_cache_nwp(source_nwp_path: str, dest_nwp_path: str):
...
         else:
             new_variables.append(var)
-    ds.__setitem__('variable', new_variables)
+    ds.__setitem__("variable", new_variables)
 
     # Save destination path
     ds.to_zarr(dest_nwp_path, mode="a")
 
 
+def set_night_time_zeros(batch, preds, sun_elevation_limit=0.0):
```
Comment (peterdudfield) [inline india_forecast_app/models/pvnet/utils.py:110]: this is new

File: tests/models/pvnet/test_utils.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+""" Tests for utils for pvnet"""
+import numpy as np
+from ocf_datapipes.batch import BatchKey
+
+from india_forecast_app.models.pvnet.utils import set_night_time_zeros
+
+
+def test_set_night_time_zeros():
+    """ Test for setting night time zeros"""
+    # set up preds (1,14,7)
```
Comment (Sukh-P) [inline tests/models/pvnet/test_utils.py:10]: Small nit, is this comment slightly out of date?

**Review by Sukh-P — APPROVED** [review]
LGTM!

File: tests/models/pvnet/test_utils.py (hunk 4 above)
Comment (peterdudfield) [reply tests/models/pvnet/test_utils.py:10]: oh yes

**Pull request merged**